source/opt/decoration_manager.h

```diff
@@ -50,6 +50,8 @@ class DecorationManager {
   bool AreDecorationsTheSame(const ir::Instruction* inst1,
                              const ir::Instruction* inst2) const;
 
+  const ir::Instruction* GetDecoration(uint32_t id, uint32_t decoration) const;
```

Comment:
Documentation?

Reply:
done.  I went with the ForEach method, but used a shorter name, and added a comment.